### ISSUE TITLE
SEO: "search and replace" 

### ISSUE THREAD

**Issue by IndianBoy42** [nvim-pack/nvim-spectre#154]
Tldr: the phrase "search and replace" should feature in the README, tags. As well as perhaps "global" and "project wide" 

So this is a weird issue to create since it's not actually technical but tbh this is the best global search and replace plugin for neovim. So it's weird that I've seen multiple Reddit posts/comments saying they don't know any good search and replace plugin for neovim, which i directed them here of course. 

So I tried to Google "neovim search and replace", which I assume most would do, and spectre is not visible. Adding global and project doesn't work either. Adding "site:github" it's still pretty low (below the wishlist post that does mention it pretty far down).

It's an open source projects so marketing is not really a Thing people are usually concerned with but I think it would be doing the community a favour. I think the key phrase "search and replace" atleast, or a bit more elaborated description, perhaps a reference to the similarity to the vscode gui, all of this could help.

**Comment by EpsilonKu** [comment]
Yeah, cool idea. But I have problem with grammar. Will be good it someone PR it with proper description.

**Comment by IndianBoy42** [comment]
Give me an hour to get back to my desktop and I can help out for sure
________________________________
